File: samples/Infralution.Localization.Wpf/SampleApp_CS/SampleEnum.cs
#region File Header
//
// COPYRIGHT:   Copyright 2009 
//              Infralution
//
#endregion
using System;
using System.Collections.Generic;
using System.Text;
using System.ComponentModel;
namespace WpfApp
{
    /// <summary>
    /// Sample enum illustrating used of a localized enum type converter
    /// </summary>
    [TypeConverter(typeof(LocalizedEnumConverter))]
    public enum SampleEnum
    {
        VerySmall,
        Small,
        Medium,
        Large,
        VeryLarge
    }

}

File: samples/Infralution.Localization.Wpf/SampleApp_VB/SampleEnum.vb
#Region "File Header"
'
' COPYRIGHT:   Copyright 2009 
'              Infralution
'
#End Region
Imports System.ComponentModel

'' <summary>
'' Sample enum illustrating used of a localized enum type converter
'' </summary>
<TypeConverter(GetType(LocalizedEnumConverter))> _
Public Enum SampleEnum
    VerySmall
    Small
    Medium
    Large
    VeryLarge
End Enum
